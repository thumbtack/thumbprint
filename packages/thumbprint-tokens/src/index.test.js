const JSZip = require('jszip');
const fs = require('fs');
const path = require('path');

const distPath = path.join(__dirname, '../dist');

it('outputted CommonJS matches snapshot', () => {
    const output = fs.readFileSync(path.join(distPath, '/javascript-cjs/index.js'), 'utf-8');
    expect(output).toMatchSnapshot();
});

it('outputted ES module matches snapshot', () => {
    const output = fs.readFileSync(path.join(distPath, 'javascript-es/index.js'), 'utf-8');
    expect(output).toMatchSnapshot();
});

it('outputted SCSS matches snapshot', () => {
    const output = fs.readFileSync(path.join(distPath, 'scss/_index.scss'), 'utf-8');
    expect(output).toMatchSnapshot();
});

it('outputted Android matches snapshot', () => {
    const output = fs.readFileSync(path.join(distPath, 'android/index.xml'), 'utf-8');
    expect(output).toMatchSnapshot();
});

it('outputted iOS matches snapshot', async () => {
    // The output is stored as a `zip`, but we unzip it for the purpose of this test so that we
    // can compare the token file as a string.
    const zipBuffer = fs.readFileSync(path.join(distPath, 'ios.zip'));
    const files = await JSZip.loadAsync(zipBuffer);
    const output = await files.file('ThumbprintTokens.swift').async('string');

    expect(output).toMatchSnapshot();
});
