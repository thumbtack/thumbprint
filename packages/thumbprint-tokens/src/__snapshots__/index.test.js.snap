// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputted Android matches snapshot 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
<resources>
    <color name=\\"tp_blue_100\\">#eaf6fa</color>
    <color name=\\"tp_blue_200\\">#b3ebff</color>
    <color name=\\"tp_blue_300\\">#79d2f2</color>
    <color name=\\"tp_blue\\">#009fd9</color>
    <color name=\\"tp_blue_500\\">#007fad</color>
    <color name=\\"tp_blue_600\\">#005a7a</color>
    <color name=\\"tp_indigo_100\\">#e8f1fd</color>
    <color name=\\"tp_indigo_200\\">#cce1ff</color>
    <color name=\\"tp_indigo_300\\">#96c2ff</color>
    <color name=\\"tp_indigo\\">#5968e2</color>
    <color name=\\"tp_indigo_500\\">#4f54b3</color>
    <color name=\\"tp_indigo_600\\">#383c80</color>
    <color name=\\"tp_purple_100\\">#f5efff</color>
    <color name=\\"tp_purple_200\\">#dfccff</color>
    <color name=\\"tp_purple_300\\">#d3baff</color>
    <color name=\\"tp_purple\\">#a97ff0</color>
    <color name=\\"tp_purple_500\\">#8a5ed6</color>
    <color name=\\"tp_purple_600\\">#502095</color>
    <color name=\\"tp_green_100\\">#e1fdf3</color>
    <color name=\\"tp_green_200\\">#c6f7da</color>
    <color name=\\"tp_green_300\\">#73e4a2</color>
    <color name=\\"tp_green\\">#2db783</color>
    <color name=\\"tp_green_500\\">#109e78</color>
    <color name=\\"tp_green_600\\">#0f6648</color>
    <color name=\\"tp_yellow_100\\">#fdf7e7</color>
    <color name=\\"tp_yellow_200\\">#ffebb3</color>
    <color name=\\"tp_yellow_300\\">#ffdd80</color>
    <color name=\\"tp_yellow\\">#febe14</color>
    <color name=\\"tp_yellow_500\\">#d48300</color>
    <color name=\\"tp_yellow_600\\">#8a5500</color>
    <color name=\\"tp_red_100\\">#ffeff0</color>
    <color name=\\"tp_red_200\\">#ffd9d9</color>
    <color name=\\"tp_red_300\\">#ffb0b0</color>
    <color name=\\"tp_red\\">#ff5a5f</color>
    <color name=\\"tp_red_500\\">#cc3553</color>
    <color name=\\"tp_red_600\\">#76012c</color>
    <color name=\\"tp_black_300\\">#676d73</color>
    <color name=\\"tp_black\\">#2f3033</color>
    <color name=\\"tp_gray_200\\">#fafafa</color>
    <color name=\\"tp_gray_300\\">#e9eced</color>
    <color name=\\"tp_gray\\">#d3d4d5</color>
    <color name=\\"tp_white\\">#ffffff</color>
    <dimen name=\\"tp_space_1\\">4dp</dimen>
    <dimen name=\\"tp_space_2\\">8dp</dimen>
    <dimen name=\\"tp_space_3\\">16dp</dimen>
    <dimen name=\\"tp_space_4\\">24dp</dimen>
    <dimen name=\\"tp_space_5\\">32dp</dimen>
    <dimen name=\\"tp_space_6\\">48dp</dimen>
</resources>
"
`;

exports[`outputted CommonJS matches snapshot 1`] = `
"\\"use strict\\";
Object.defineProperty(exports, \\"__esModule\\", { value: true });
exports.tpBorderRadiusBase = \\"4px\\";
exports.tpBorderRadiusBig = \\"6px\\";
exports.tpBorderRadiusFull = \\"50%\\";
exports.tpBorderRadiusSides = \\"9999px\\";
exports.tpBreakpointLarge = \\"1025px\\";
exports.tpBreakpointLargeValue = 1025;
exports.tpBreakpointMedium = \\"700px\\";
exports.tpBreakpointMediumValue = 700;
exports.tpBreakpointSmall = \\"481px\\";
exports.tpBreakpointSmallValue = 481;
exports.tpBreakpointSplitViewMedium = \\"1060px\\";
exports.tpBreakpointSplitViewMediumValue = 1060;
exports.tpBreakpointSplitViewSmall = \\"769px\\";
exports.tpBreakpointSplitViewSmallValue = 769;
exports.tpColorBlue100 = \\"#eaf6fa\\";
exports.tpColorBlue200 = \\"#b3ebff\\";
exports.tpColorBlue300 = \\"#79d2f2\\";
exports.tpColorBlue = \\"#009fd9\\";
exports.tpColorBlue500 = \\"#007fad\\";
exports.tpColorBlue600 = \\"#005a7a\\";
exports.tpColorIndigo100 = \\"#e8f1fd\\";
exports.tpColorIndigo200 = \\"#cce1ff\\";
exports.tpColorIndigo300 = \\"#96c2ff\\";
exports.tpColorIndigo = \\"#5968e2\\";
exports.tpColorIndigo500 = \\"#4f54b3\\";
exports.tpColorIndigo600 = \\"#383c80\\";
exports.tpColorPurple100 = \\"#f5efff\\";
exports.tpColorPurple200 = \\"#dfccff\\";
exports.tpColorPurple300 = \\"#d3baff\\";
exports.tpColorPurple = \\"#a97ff0\\";
exports.tpColorPurple500 = \\"#8a5ed6\\";
exports.tpColorPurple600 = \\"#502095\\";
exports.tpColorGreen100 = \\"#e1fdf3\\";
exports.tpColorGreen200 = \\"#c6f7da\\";
exports.tpColorGreen300 = \\"#73e4a2\\";
exports.tpColorGreen = \\"#2db783\\";
exports.tpColorGreen500 = \\"#109e78\\";
exports.tpColorGreen600 = \\"#0f6648\\";
exports.tpColorYellow100 = \\"#fdf7e7\\";
exports.tpColorYellow200 = \\"#ffebb3\\";
exports.tpColorYellow300 = \\"#ffdd80\\";
exports.tpColorYellow = \\"#febe14\\";
exports.tpColorYellow500 = \\"#d48300\\";
exports.tpColorYellow600 = \\"#8a5500\\";
exports.tpColorRed100 = \\"#ffeff0\\";
exports.tpColorRed200 = \\"#ffd9d9\\";
exports.tpColorRed300 = \\"#ffb0b0\\";
exports.tpColorRed = \\"#ff5a5f\\";
exports.tpColorRed500 = \\"#cc3553\\";
exports.tpColorRed600 = \\"#76012c\\";
exports.tpColorBlack300 = \\"#676d73\\";
exports.tpColorBlack = \\"#2f3033\\";
exports.tpColorGray200 = \\"#fafafa\\";
exports.tpColorGray300 = \\"#e9eced\\";
exports.tpColorGray = \\"#d3d4d5\\";
exports.tpColorWhite = \\"#ffffff\\";
exports.tpFontFamilyBase = \\"Mark, Avenir, Helvetica, Arial, sans-serif\\";
exports.tpFontFamilyMonospace = \\"'Source Code Pro', monospace\\";
exports.tpFontWeightNormal = \\"400\\";
exports.tpFontWeightBold = \\"700\\";
exports.tpFontTitle1Size = \\"28px\\";
exports.tpFontTitle1LineHeight = \\"32px\\";
exports.tpFontTitle1Weight = \\"700\\";
exports.tpFontTitle1ResponsiveSize = \\"40px\\";
exports.tpFontTitle1ResponsiveLineHeight = \\"44px\\";
exports.tpFontTitle1ResponsiveWeight = \\"700\\";
exports.tpFontTitle2Size = \\"24px\\";
exports.tpFontTitle2LineHeight = \\"28px\\";
exports.tpFontTitle2Weight = \\"700\\";
exports.tpFontTitle2ResponsiveSize = \\"32px\\";
exports.tpFontTitle2ResponsiveLineHeight = \\"40px\\";
exports.tpFontTitle2ResponsiveWeight = \\"700\\";
exports.tpFontTitle3Size = \\"22px\\";
exports.tpFontTitle3LineHeight = \\"28px\\";
exports.tpFontTitle3Weight = \\"700\\";
exports.tpFontTitle3ResponsiveSize = \\"24px\\";
exports.tpFontTitle3ResponsiveLineHeight = \\"32px\\";
exports.tpFontTitle3ResponsiveWeight = \\"700\\";
exports.tpFontTitle4Size = \\"20px\\";
exports.tpFontTitle4LineHeight = \\"28px\\";
exports.tpFontTitle4Weight = \\"700\\";
exports.tpFontTitle5Size = \\"18px\\";
exports.tpFontTitle5LineHeight = \\"24px\\";
exports.tpFontTitle5Weight = \\"700\\";
exports.tpFontTitle6Size = \\"16px\\";
exports.tpFontTitle6LineHeight = \\"24px\\";
exports.tpFontTitle6Weight = \\"700\\";
exports.tpFontTitle7Size = \\"14px\\";
exports.tpFontTitle7LineHeight = \\"20px\\";
exports.tpFontTitle7Weight = \\"700\\";
exports.tpFontTitle8Size = \\"12px\\";
exports.tpFontTitle8LineHeight = \\"18px\\";
exports.tpFontTitle8Weight = \\"700\\";
exports.tpFontBody1Size = \\"16px\\";
exports.tpFontBody1LineHeight = \\"24px\\";
exports.tpFontBody2Size = \\"14px\\";
exports.tpFontBody2LineHeight = \\"20px\\";
exports.tpFontBody3Size = \\"12px\\";
exports.tpFontBody3LineHeight = \\"18px\\";
exports.tpLetterSpacingLoose = \\"1px\\";
exports.tpLetterSpacingTight = \\"-1px\\";
exports.tpLetterSpacingExtraTight = \\"-2px\\";
exports.tpLineHeightBase = \\"1.6\\";
exports.tpLineHeightTight = \\"1.4\\";
exports.tpLineHeightLoose = \\"1.9\\";
exports.tpShadow100 = \\"0px 1px 3px rgba(0, 0, 0, 0.1)\\";
exports.tpShadow200 = \\"0px 2px 4px rgba(0, 0, 0, 0.15)\\";
exports.tpShadow300 = \\"0px 2px 7px rgba(0, 0, 0, 0.15)\\";
exports.tpShadow400 = \\"0px 2px 10px rgba(0, 0, 0, 0.2)\\";
// @deprecated
exports.tpShadowCard = \\"0 -1px 1px 0 rgba(0, 0, 0, 0.04), 0 1px 2px 0 rgba(0, 0, 0, 0.16)\\";
exports.tpSpace1 = \\"4px\\";
exports.tpSpace2 = \\"8px\\";
exports.tpSpace3 = \\"16px\\";
exports.tpSpace4 = \\"24px\\";
exports.tpSpace5 = \\"32px\\";
exports.tpSpace6 = \\"64px\\";
exports.tpSpace7 = \\"128px\\";
exports.tpSpace8 = \\"256px\\";
exports.tpWrapMaxWidth = \\"946px\\";
exports.tpWrapNoPadWidth = \\"1010px\\";
exports.tpZIndexModal = 200;
"
`;

exports[`outputted ES module matches snapshot 1`] = `
"export var tpBorderRadiusBase = \\"4px\\";
export var tpBorderRadiusBig = \\"6px\\";
export var tpBorderRadiusFull = \\"50%\\";
export var tpBorderRadiusSides = \\"9999px\\";
export var tpBreakpointLarge = \\"1025px\\";
export var tpBreakpointLargeValue = 1025;
export var tpBreakpointMedium = \\"700px\\";
export var tpBreakpointMediumValue = 700;
export var tpBreakpointSmall = \\"481px\\";
export var tpBreakpointSmallValue = 481;
export var tpBreakpointSplitViewMedium = \\"1060px\\";
export var tpBreakpointSplitViewMediumValue = 1060;
export var tpBreakpointSplitViewSmall = \\"769px\\";
export var tpBreakpointSplitViewSmallValue = 769;
export var tpColorBlue100 = \\"#eaf6fa\\";
export var tpColorBlue200 = \\"#b3ebff\\";
export var tpColorBlue300 = \\"#79d2f2\\";
export var tpColorBlue = \\"#009fd9\\";
export var tpColorBlue500 = \\"#007fad\\";
export var tpColorBlue600 = \\"#005a7a\\";
export var tpColorIndigo100 = \\"#e8f1fd\\";
export var tpColorIndigo200 = \\"#cce1ff\\";
export var tpColorIndigo300 = \\"#96c2ff\\";
export var tpColorIndigo = \\"#5968e2\\";
export var tpColorIndigo500 = \\"#4f54b3\\";
export var tpColorIndigo600 = \\"#383c80\\";
export var tpColorPurple100 = \\"#f5efff\\";
export var tpColorPurple200 = \\"#dfccff\\";
export var tpColorPurple300 = \\"#d3baff\\";
export var tpColorPurple = \\"#a97ff0\\";
export var tpColorPurple500 = \\"#8a5ed6\\";
export var tpColorPurple600 = \\"#502095\\";
export var tpColorGreen100 = \\"#e1fdf3\\";
export var tpColorGreen200 = \\"#c6f7da\\";
export var tpColorGreen300 = \\"#73e4a2\\";
export var tpColorGreen = \\"#2db783\\";
export var tpColorGreen500 = \\"#109e78\\";
export var tpColorGreen600 = \\"#0f6648\\";
export var tpColorYellow100 = \\"#fdf7e7\\";
export var tpColorYellow200 = \\"#ffebb3\\";
export var tpColorYellow300 = \\"#ffdd80\\";
export var tpColorYellow = \\"#febe14\\";
export var tpColorYellow500 = \\"#d48300\\";
export var tpColorYellow600 = \\"#8a5500\\";
export var tpColorRed100 = \\"#ffeff0\\";
export var tpColorRed200 = \\"#ffd9d9\\";
export var tpColorRed300 = \\"#ffb0b0\\";
export var tpColorRed = \\"#ff5a5f\\";
export var tpColorRed500 = \\"#cc3553\\";
export var tpColorRed600 = \\"#76012c\\";
export var tpColorBlack300 = \\"#676d73\\";
export var tpColorBlack = \\"#2f3033\\";
export var tpColorGray200 = \\"#fafafa\\";
export var tpColorGray300 = \\"#e9eced\\";
export var tpColorGray = \\"#d3d4d5\\";
export var tpColorWhite = \\"#ffffff\\";
export var tpFontFamilyBase = \\"Mark, Avenir, Helvetica, Arial, sans-serif\\";
export var tpFontFamilyMonospace = \\"'Source Code Pro', monospace\\";
export var tpFontWeightNormal = \\"400\\";
export var tpFontWeightBold = \\"700\\";
export var tpFontTitle1Size = \\"28px\\";
export var tpFontTitle1LineHeight = \\"32px\\";
export var tpFontTitle1Weight = \\"700\\";
export var tpFontTitle1ResponsiveSize = \\"40px\\";
export var tpFontTitle1ResponsiveLineHeight = \\"44px\\";
export var tpFontTitle1ResponsiveWeight = \\"700\\";
export var tpFontTitle2Size = \\"24px\\";
export var tpFontTitle2LineHeight = \\"28px\\";
export var tpFontTitle2Weight = \\"700\\";
export var tpFontTitle2ResponsiveSize = \\"32px\\";
export var tpFontTitle2ResponsiveLineHeight = \\"40px\\";
export var tpFontTitle2ResponsiveWeight = \\"700\\";
export var tpFontTitle3Size = \\"22px\\";
export var tpFontTitle3LineHeight = \\"28px\\";
export var tpFontTitle3Weight = \\"700\\";
export var tpFontTitle3ResponsiveSize = \\"24px\\";
export var tpFontTitle3ResponsiveLineHeight = \\"32px\\";
export var tpFontTitle3ResponsiveWeight = \\"700\\";
export var tpFontTitle4Size = \\"20px\\";
export var tpFontTitle4LineHeight = \\"28px\\";
export var tpFontTitle4Weight = \\"700\\";
export var tpFontTitle5Size = \\"18px\\";
export var tpFontTitle5LineHeight = \\"24px\\";
export var tpFontTitle5Weight = \\"700\\";
export var tpFontTitle6Size = \\"16px\\";
export var tpFontTitle6LineHeight = \\"24px\\";
export var tpFontTitle6Weight = \\"700\\";
export var tpFontTitle7Size = \\"14px\\";
export var tpFontTitle7LineHeight = \\"20px\\";
export var tpFontTitle7Weight = \\"700\\";
export var tpFontTitle8Size = \\"12px\\";
export var tpFontTitle8LineHeight = \\"18px\\";
export var tpFontTitle8Weight = \\"700\\";
export var tpFontBody1Size = \\"16px\\";
export var tpFontBody1LineHeight = \\"24px\\";
export var tpFontBody2Size = \\"14px\\";
export var tpFontBody2LineHeight = \\"20px\\";
export var tpFontBody3Size = \\"12px\\";
export var tpFontBody3LineHeight = \\"18px\\";
export var tpLetterSpacingLoose = \\"1px\\";
export var tpLetterSpacingTight = \\"-1px\\";
export var tpLetterSpacingExtraTight = \\"-2px\\";
export var tpLineHeightBase = \\"1.6\\";
export var tpLineHeightTight = \\"1.4\\";
export var tpLineHeightLoose = \\"1.9\\";
export var tpShadow100 = \\"0px 1px 3px rgba(0, 0, 0, 0.1)\\";
export var tpShadow200 = \\"0px 2px 4px rgba(0, 0, 0, 0.15)\\";
export var tpShadow300 = \\"0px 2px 7px rgba(0, 0, 0, 0.15)\\";
export var tpShadow400 = \\"0px 2px 10px rgba(0, 0, 0, 0.2)\\";
// @deprecated
export var tpShadowCard = \\"0 -1px 1px 0 rgba(0, 0, 0, 0.04), 0 1px 2px 0 rgba(0, 0, 0, 0.16)\\";
export var tpSpace1 = \\"4px\\";
export var tpSpace2 = \\"8px\\";
export var tpSpace3 = \\"16px\\";
export var tpSpace4 = \\"24px\\";
export var tpSpace5 = \\"32px\\";
export var tpSpace6 = \\"64px\\";
export var tpSpace7 = \\"128px\\";
export var tpSpace8 = \\"256px\\";
export var tpWrapMaxWidth = \\"946px\\";
export var tpWrapNoPadWidth = \\"1010px\\";
export var tpZIndexModal = 200;
"
`;

exports[`outputted SCSS matches snapshot 1`] = `
"$tp-border-radius__base: 4px;
$tp-border-radius__big: 6px;
$tp-border-radius__full: 50%;
$tp-border-radius__sides: 9999px;
$tp-breakpoint__large: 1025px;
$tp-breakpoint__large__value: 1025;
$tp-breakpoint__medium: 700px;
$tp-breakpoint__medium__value: 700;
$tp-breakpoint__small: 481px;
$tp-breakpoint__small__value: 481;
$tp-breakpoint__split-view__medium: 1060px;
$tp-breakpoint__split-view__medium__value: 1060;
$tp-breakpoint__split-view__small: 769px;
$tp-breakpoint__split-view__small__value: 769;
$tp-color__blue-100: #eaf6fa;
$tp-color__blue-200: #b3ebff;
$tp-color__blue-300: #79d2f2;
$tp-color__blue: #009fd9;
$tp-color__blue-500: #007fad;
$tp-color__blue-600: #005a7a;
$tp-color__indigo-100: #e8f1fd;
$tp-color__indigo-200: #cce1ff;
$tp-color__indigo-300: #96c2ff;
$tp-color__indigo: #5968e2;
$tp-color__indigo-500: #4f54b3;
$tp-color__indigo-600: #383c80;
$tp-color__purple-100: #f5efff;
$tp-color__purple-200: #dfccff;
$tp-color__purple-300: #d3baff;
$tp-color__purple: #a97ff0;
$tp-color__purple-500: #8a5ed6;
$tp-color__purple-600: #502095;
$tp-color__green-100: #e1fdf3;
$tp-color__green-200: #c6f7da;
$tp-color__green-300: #73e4a2;
$tp-color__green: #2db783;
$tp-color__green-500: #109e78;
$tp-color__green-600: #0f6648;
$tp-color__yellow-100: #fdf7e7;
$tp-color__yellow-200: #ffebb3;
$tp-color__yellow-300: #ffdd80;
$tp-color__yellow: #febe14;
$tp-color__yellow-500: #d48300;
$tp-color__yellow-600: #8a5500;
$tp-color__red-100: #ffeff0;
$tp-color__red-200: #ffd9d9;
$tp-color__red-300: #ffb0b0;
$tp-color__red: #ff5a5f;
$tp-color__red-500: #cc3553;
$tp-color__red-600: #76012c;
$tp-color__black-300: #676d73;
$tp-color__black: #2f3033;
$tp-color__gray-200: #fafafa;
$tp-color__gray-300: #e9eced;
$tp-color__gray: #d3d4d5;
$tp-color__white: #ffffff;
$tp-font-family__base: Mark, Avenir, Helvetica, Arial, sans-serif;
$tp-font-family__monospace: 'Source Code Pro', monospace;
$tp-font-weight__normal: 400;
$tp-font-weight__bold: 700;
$tp-font__title__1__size: 28px;
$tp-font__title__1__line-height: 32px;
$tp-font__title__1__weight: 700;
$tp-font__title__1__responsive__size: 40px;
$tp-font__title__1__responsive__line-height: 44px;
$tp-font__title__1__responsive__weight: 700;
$tp-font__title__2__size: 24px;
$tp-font__title__2__line-height: 28px;
$tp-font__title__2__weight: 700;
$tp-font__title__2__responsive__size: 32px;
$tp-font__title__2__responsive__line-height: 40px;
$tp-font__title__2__responsive__weight: 700;
$tp-font__title__3__size: 22px;
$tp-font__title__3__line-height: 28px;
$tp-font__title__3__weight: 700;
$tp-font__title__3__responsive__size: 24px;
$tp-font__title__3__responsive__line-height: 32px;
$tp-font__title__3__responsive__weight: 700;
$tp-font__title__4__size: 20px;
$tp-font__title__4__line-height: 28px;
$tp-font__title__4__weight: 700;
$tp-font__title__5__size: 18px;
$tp-font__title__5__line-height: 24px;
$tp-font__title__5__weight: 700;
$tp-font__title__6__size: 16px;
$tp-font__title__6__line-height: 24px;
$tp-font__title__6__weight: 700;
$tp-font__title__7__size: 14px;
$tp-font__title__7__line-height: 20px;
$tp-font__title__7__weight: 700;
$tp-font__title__8__size: 12px;
$tp-font__title__8__line-height: 18px;
$tp-font__title__8__weight: 700;
$tp-font__body__1__size: 16px;
$tp-font__body__1__line-height: 24px;
$tp-font__body__2__size: 14px;
$tp-font__body__2__line-height: 20px;
$tp-font__body__3__size: 12px;
$tp-font__body__3__line-height: 18px;
$tp-letter-spacing__loose: 1px;
$tp-letter-spacing__tight: -1px;
$tp-letter-spacing__extra-tight: -2px;
$tp-line-height__base: 1.6;
$tp-line-height__tight: 1.4;
$tp-line-height__loose: 1.9;
$tp-shadow__100: 0px 1px 3px rgba(0, 0, 0, 0.1);
$tp-shadow__200: 0px 2px 4px rgba(0, 0, 0, 0.15);
$tp-shadow__300: 0px 2px 7px rgba(0, 0, 0, 0.15);
$tp-shadow__400: 0px 2px 10px rgba(0, 0, 0, 0.2);
// @deprecated
$tp-shadow__card: 0 -1px 1px 0 rgba(0, 0, 0, 0.04), 0 1px 2px 0 rgba(0, 0, 0, 0.16);
$tp-space__1: 4px;
$tp-space__2: 8px;
$tp-space__3: 16px;
$tp-space__4: 24px;
$tp-space__5: 32px;
$tp-space__6: 64px;
$tp-space__7: 128px;
$tp-space__8: 256px;
$tp-wrap__max-width: 946px;
$tp-wrap__no-pad-width: 1010px;
$tp-z-index__modal: 200;
"
`;

exports[`outputted iOS matches snapshot 1`] = `
"import UIKit.UIColor

/// These are the colors we use at Thumbtack.
public enum Color {
    /// Blue 100 – #eaf6fa
    public static let blue100: UIColor = UIColor(red: 0.91764706, green: 0.9647059, blue: 0.98039216, alpha: 1.0)
    /// Blue 200 – #b3ebff
    public static let blue200: UIColor = UIColor(red: 0.7019608, green: 0.92156863, blue: 1.0, alpha: 1.0)
    /// Blue 300 – #79d2f2
    public static let blue300: UIColor = UIColor(red: 0.4745098, green: 0.8235294, blue: 0.9490196, alpha: 1.0)
    /// Blue – #009fd9
    public static let blue: UIColor = UIColor(red: 0.0, green: 0.62352943, blue: 0.8509804, alpha: 1.0)
    /// Blue 500 – #007fad
    public static let blue500: UIColor = UIColor(red: 0.0, green: 0.49803922, blue: 0.6784314, alpha: 1.0)
    /// Blue 600 – #005a7a
    public static let blue600: UIColor = UIColor(red: 0.0, green: 0.3529412, blue: 0.47843137, alpha: 1.0)
    /// Indigo 100 – #e8f1fd
    public static let indigo100: UIColor = UIColor(red: 0.9098039, green: 0.94509804, blue: 0.99215686, alpha: 1.0)
    /// Indigo 200 – #cce1ff
    public static let indigo200: UIColor = UIColor(red: 0.8, green: 0.88235295, blue: 1.0, alpha: 1.0)
    /// Indigo 300 – #96c2ff
    public static let indigo300: UIColor = UIColor(red: 0.5882353, green: 0.7607843, blue: 1.0, alpha: 1.0)
    /// Indigo – #5968e2
    public static let indigo: UIColor = UIColor(red: 0.34901962, green: 0.40784314, blue: 0.8862745, alpha: 1.0)
    /// Indigo 500 – #4f54b3
    public static let indigo500: UIColor = UIColor(red: 0.30980393, green: 0.32941177, blue: 0.7019608, alpha: 1.0)
    /// Indigo 600 – #383c80
    public static let indigo600: UIColor = UIColor(red: 0.21960784, green: 0.23529412, blue: 0.5019608, alpha: 1.0)
    /// Purple 100 – #f5efff
    public static let purple100: UIColor = UIColor(red: 0.9607843, green: 0.9372549, blue: 1.0, alpha: 1.0)
    /// Purple 200 – #dfccff
    public static let purple200: UIColor = UIColor(red: 0.8745098, green: 0.8, blue: 1.0, alpha: 1.0)
    /// Purple 300 – #d3baff
    public static let purple300: UIColor = UIColor(red: 0.827451, green: 0.7294118, blue: 1.0, alpha: 1.0)
    /// Purple – #a97ff0
    public static let purple: UIColor = UIColor(red: 0.6627451, green: 0.49803922, blue: 0.9411765, alpha: 1.0)
    /// Purple 500 – #8a5ed6
    public static let purple500: UIColor = UIColor(red: 0.5411765, green: 0.36862746, blue: 0.8392157, alpha: 1.0)
    /// Purple 600 – #502095
    public static let purple600: UIColor = UIColor(red: 0.3137255, green: 0.1254902, blue: 0.58431375, alpha: 1.0)
    /// Green 100 – #e1fdf3
    public static let green100: UIColor = UIColor(red: 0.88235295, green: 0.99215686, blue: 0.9529412, alpha: 1.0)
    /// Green 200 – #c6f7da
    public static let green200: UIColor = UIColor(red: 0.7764706, green: 0.96862745, blue: 0.85490197, alpha: 1.0)
    /// Green 300 – #73e4a2
    public static let green300: UIColor = UIColor(red: 0.4509804, green: 0.89411765, blue: 0.63529414, alpha: 1.0)
    /// Green – #2db783
    public static let green: UIColor = UIColor(red: 0.1764706, green: 0.7176471, blue: 0.5137255, alpha: 1.0)
    /// Green 500 – #109e78
    public static let green500: UIColor = UIColor(red: 0.0627451, green: 0.61960787, blue: 0.47058824, alpha: 1.0)
    /// Green 600 – #0f6648
    public static let green600: UIColor = UIColor(red: 0.05882353, green: 0.4, blue: 0.28235295, alpha: 1.0)
    /// Yellow 100 – #fdf7e7
    public static let yellow100: UIColor = UIColor(red: 0.99215686, green: 0.96862745, blue: 0.90588236, alpha: 1.0)
    /// Yellow 200 – #ffebb3
    public static let yellow200: UIColor = UIColor(red: 1.0, green: 0.92156863, blue: 0.7019608, alpha: 1.0)
    /// Yellow 300 – #ffdd80
    public static let yellow300: UIColor = UIColor(red: 1.0, green: 0.8666667, blue: 0.5019608, alpha: 1.0)
    /// Yellow – #febe14
    public static let yellow: UIColor = UIColor(red: 0.99607843, green: 0.74509805, blue: 0.078431375, alpha: 1.0)
    /// Yellow 500 – #d48300
    public static let yellow500: UIColor = UIColor(red: 0.83137256, green: 0.5137255, blue: 0.0, alpha: 1.0)
    /// Yellow 600 – #8a5500
    public static let yellow600: UIColor = UIColor(red: 0.5411765, green: 0.33333334, blue: 0.0, alpha: 1.0)
    /// Red 100 – #ffeff0
    public static let red100: UIColor = UIColor(red: 1.0, green: 0.9372549, blue: 0.9411765, alpha: 1.0)
    /// Red 200 – #ffd9d9
    public static let red200: UIColor = UIColor(red: 1.0, green: 0.8509804, blue: 0.8509804, alpha: 1.0)
    /// Red 300 – #ffb0b0
    public static let red300: UIColor = UIColor(red: 1.0, green: 0.6901961, blue: 0.6901961, alpha: 1.0)
    /// Red – #ff5a5f
    public static let red: UIColor = UIColor(red: 1.0, green: 0.3529412, blue: 0.37254903, alpha: 1.0)
    /// Red 500 – #cc3553
    public static let red500: UIColor = UIColor(red: 0.8, green: 0.20784314, blue: 0.3254902, alpha: 1.0)
    /// Red 600 – #76012c
    public static let red600: UIColor = UIColor(red: 0.4627451, green: 0.003921569, blue: 0.17254902, alpha: 1.0)
    /// Black 300 – #676d73
    public static let black300: UIColor = UIColor(red: 0.40392157, green: 0.42745098, blue: 0.4509804, alpha: 1.0)
    /// Black – #2f3033
    public static let black: UIColor = UIColor(red: 0.18431373, green: 0.1882353, blue: 0.2, alpha: 1.0)
    /// Gray 200 – #fafafa
    public static let gray200: UIColor = UIColor(red: 0.98039216, green: 0.98039216, blue: 0.98039216, alpha: 1.0)
    /// Gray 300 – #e9eced
    public static let gray300: UIColor = UIColor(red: 0.9137255, green: 0.9254902, blue: 0.92941177, alpha: 1.0)
    /// Gray – #d3d4d5
    public static let gray: UIColor = UIColor(red: 0.827451, green: 0.83137256, blue: 0.8352941, alpha: 1.0)
    /// White – #ffffff
    public static let white: UIColor = UIColor(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
}

public enum Space {
    /// Space 1 – 4.0pt
    public static let one: CGFloat = 4.0
    /// Space 2 – 8.0pt
    public static let two: CGFloat = 8.0
    /// Space 3 – 16.0pt
    public static let three: CGFloat = 16.0
    /// Space 4 – 24.0pt
    public static let four: CGFloat = 24.0
    /// Space 5 – 32.0pt
    public static let five: CGFloat = 32.0
    /// Space 6 – 48.0pt
    public static let six: CGFloat = 48.0
}
"
`;
